### PR TITLE
disable approve quote for admin & fix image viewer styling

### PR DESCRIPTION
**Pull request by samsep**

### CODE REVIEW

File: src/scripts/timeline.controller.coffee [hunk 1]
```diff
@@ -12,22 +13,23 @@ TimelineController = ($scope, $stateParams, $document, TimelineAPIService, Copil
   vm.expanded = {}
 
   vm.acceptQuote = (event) ->
-    if vm.copilot?.userId
-      params =
-        userId: vm.copilot.userId
-        projectId: vm.workId
+    if vm.userType != 'admin'
```
Comment (atranappirio) [inline src/scripts/timeline.controller.coffee]: I dont think this is needed

File: src/scripts/timeline.controller.coffee [hunk 2]
```diff
@@ -3,6 +3,7 @@
 TimelineController = ($scope, $stateParams, $document, TimelineAPIService, CopilotApprovalAPIService) ->
   vm                       = this
   vm.eventGroups           = []
+  vm.userType              = $scope.userType || 'customer'
```
Comment (atranappirio) [inline src/scripts/timeline.controller.coffee]: should be permission base: POST, DELETE, PUT. (POST wont be valid for timeline)

File: src/views/parts/quote.jade [hunk 3]
```diff
@@ -35,6 +35,6 @@ event-card-main.transition(
 
         h6 Time estimate
 
-  button.action(ng-click="vm.acceptQuote()" ng-if="vm.showAcceptQuoteButton") approve quote
+  button.action(ng-click="vm.acceptQuote()" ng-if="vm.showAcceptQuoteButton" ng-disabled="vm.permissions.indexOf('CREATE') == -1") approve quote
```
Comment (vietnogi) [inline src/views/parts/quote.jade]: Would this be create or update?

Comment (samsep) [inline src/views/parts/quote.jade]: The actual call is a POST so it's one of those ambiguous ones

**Comment by vietnogi** [comment]
LGTM

**Pull request merged**